app/src/main/java/org/fossasia/pslab/activity/AboutUs.java

```diff
@@ -17,7 +20,40 @@
     @Override
     protected void onCreate(@Nullable Bundle savedInstanceState) {
         super.onCreate(savedInstanceState);
-        setContentView(R.layout.activity_about_us);
-        ButterKnife.bind(this);
+        simulateDayNight(0);
+        View aboutPage = new AboutPage(this)
+                .isRTL(false)
+                .setImage(R.drawable.logo200x200)
+                .setDescription(getString(R.string.about_us_description))
+                .addGroup("Connect with us")
+                .addEmail("pslab-fossasia@googlegroups.com")
+                .addWebsite("http://pslab.fossasia.org/")
+                .addGitHub("fossasia")
```

Comment:
This is soo cool! Should we add links to repositories and other pages inside these brackets? :)